### PR TITLE
DOCS-2176 ARM Agent Release Notes

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: release-notes/v/latest/runtime-manager-agent-1.9.5-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+= Runtime Manager Agent 1.9.5 Release Notes
+
+_May 5, 2018_
+
+This document describes new features and enhancements, known limitations, issues, and fixes in Anypoint Runtime Manager Agent, version 1.9.5.
+
+== New Features and Enhancements
+
+* Added log entries for web socket disconnections to help understand the root cause of server disconnections.
```
Comment (Bardammu) [inline release-notes/v/latest/runtime-manager-agent-1.9.5-release-notes.adoc]: Theses lines should be removed.

File: release-notes/v/latest/runtime-manager-agent-1.9.5-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+* Reduced logging verbosity of the Runtime Manager Agent to avoid unnecessary log noise
```
Comment (Bardammu) [inline release-notes/v/latest/runtime-manager-agent-1.9.5-release-notes.adoc]: This should be removed as well

**Comment by sleslie23** [comment]
@Bardammu Thanks! Can you verify?

**Comment by sleslie23** [comment]
Thanks @Bardammu !!!

**Pull request merged**